site/content/en/docs/manual/advanced/cli.md

```diff
@@ -21,7 +22,24 @@ Overview of functionality:
 - Export and download a whole task
 - Import a task
 
-**Usage**
+CVAT Command line interface use `argparse`.
```

Comment:
@TOsmanov , it is an implement detail. I don't think that we need the information.